internal/relay/ses/relay.go

```diff
@@ -16,6 +16,9 @@ type Client struct {
 	setName         *string
 	allowFromRegExp *regexp.Regexp
 	denyToRegExp    *regexp.Regexp
+	sourceArn       *string
+	fromArn         *string
+	returnPathArn   *string
```

Comment:
I think it might be better to create a struct called `arns` that consists of those three arns and add that as member to this struct. Makes handling the arguments for a lot of the function calls a bit simpler.